clang/lib/Sema/SemaSYCL.cpp

```diff
@@ -4260,6 +4379,17 @@ void Sema::ConstructOpenCLKernel(FunctionDecl *KernelCallerFunc,
     if (!getLangOpts().OptRecordFile.empty())
       opt_report.handleSyclKernelHandlerType();
   }
+  /// The Aie runtime has different code pattern form intel's runtime,
+  /// This is the way we generate inclusion headers for the Aie runtime.
```

Comment:
```suggestion
  /// The AIE runtime has different code patterns from Intel's runtime,
  /// This is the way we generate inclusion headers for the AIE runtime.
```
"This is the way..." Not super explicit. Is it described somewhere?